src/server/routes/parcel.routes.spec.ts

```diff
@@ -152,4 +161,85 @@ describe('parcel route', () => {
     expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.ACCEPTED);
     expect(logs).toContainEqual(partialPinoLog('info', 'Ignoring invalid service message'));
   });
+
+  describe('store PDA', () => {
+    test('Valid PDA should be stored', async () => {
+      const certificatePath = await generatePDACertificationPath(KEY_PAIR_SET);
```

Comment:
We can reuse this `certificatePath` in all the other tests, so let's do that by defining it in a `beforeAll()` (preferably) or `beforeEach()`.